### ISSUE TITLE
Does this model support training with a 48k sampling rate?

### ISSUE THREAD

**Issue by WangQin-wq** [starrytong/SCNet#34]
I converted the dataset to 48k and modified the code, and it can train normally, but the results on the test set are incorrect

<img width="822" height="562" alt="Image" src="https://github.com/user-attachments/assets/9c236ce1-8be1-42e1-98e1-85820614374a" />

**Comment by DM-Rookie** [comment]
It should be supported in theory. Are the loss and SDR on the validation set looking good for 48kHz compared to 44.1kHz? If so, take a look at the inference part. The current code downsamples everything to 44100.
 ```
        # Convert to model's expected format
        mix = convert_audio(mix, sample_rate, 44100, self.separator.audio_channels)
 ```
